tests/k8s/1-020_validate_redis_ha_nonha_services/01-assert.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 720
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: example-argocd-redis
```

Comment:
Please also check for `.status.phase` of the `ArgoCD` object to be up & running here